exporter/datadogexporter/internal/testutil/logs.go

```diff
@@ -90,3 +93,32 @@ func gUnzipData(rg io.Reader) ([]byte, error) {
 	}
 	return io.ReadAll(r)
 }
+
+// MockLogsEndpoint returns the processed JSON log data for each endpoint call
+func MockLogsEndpoint(w http.ResponseWriter, r *http.Request) JsonLogs {
+	// we can reuse same response object for logs as well
+	res := metricsResponse{Status: "ok"}
+	resJSON, err := json.Marshal(res)
+	if err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+		log.Fatalln(err)
+	}
```

Comment:
This is very out of place (it's used all the way down). Why not simply do:
```go
if _, err := w.Write([]byte(`{"status":"ok"}`); err != nil {
```
Why bother with all the marshalling and types etc?